### PR TITLE
Use Registry.redhat.io base images in builds

### PR DESCRIPTION
**Pull request by sbose78**
image

image

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_registry_redhat_io.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Configure the Application/Component to use the above token.
+
+
+. On the "Component Settings" page, click add a new *Build Secret*.
+. In the *Create new Build Secret* modal, add the registry.redhat.io token to be used as a pull secret:
+..   Enter *.dockerconfigjson*. in the  *Key* input field.
+..   Enter *registry-redhat-io* in the *Select or Enter Name* input field.
+..   Copy-Paste the registry service account token retrieved in the previous.
+..  Click *Create*.
+.. From the command-line, run "oc secrets link appstudio-pipeline registry-redhat-io".
```
Comment (scoheb) [inline docs/modules/ROOT/pages/how-to-guides/Import-code/proc_registry_redhat_io.adoc:32]: normal users cannot run this command...

Comment (sbose78) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_registry_redhat_io.adoc:32]: A recent bug broke this https://issues.redhat.com/browse/RHTAPBUGS-338 , fix https://github.com/codeready-toolchain/host-operator/pull/811

**Review by scoheb — APPROVED** [review]

File: docs/modules/ROOT/pages/how-to-guides/Import-code/proc_registry_redhat_io.adoc (hunk 1 above)
Comment (adambkaplan) [inline docs/modules/ROOT/pages/how-to-guides/Import-code/proc_registry_redhat_io.adoc:32]: Can we find a way where a user doesn't have to do this, and do we have a story/feature tracking this?

Comment (scoheb) [reply docs/modules/ROOT/pages/how-to-guides/Import-code/proc_registry_redhat_io.adoc:32]: Actually I don't even think this command is needed. When I was investigating adding the snyk token, it seems like HAC is already interacting with SPI and the secret is being linked.

**Comment by jc-berger** [comment]
Docs has made a PR including these changes and some more doc edits: https://github.com/redhat-appstudio/docs.appstudio.io/pull/122

**Pull request closed without merging**